source/patterns/@aws-solutions-constructs/aws-dynamodb-stream-lambda/lib/index.ts

```diff
@@ -90,11 +91,12 @@ export class DynamoDBStreamToLambda extends Construct {
 
     const table: dynamodb.ITable = defaults.buildDynamoDBTableWithStream(this, {
```

Comment:
Purposefully pointing to an object as the wrong type seems a little cheesy (and fraught with peril). I like the returning 2 values approach better. It also allows cleaner code in this routine, as this line could be"

```
[dynamoTableInterface, dynamoTable] = defaults.buildDynamoDBTableWithStream(...
```

Lines 97-99 are then no longer needed